frontend/portals/SearchSuggestions/index.jsx

```diff
@@ -16,7 +19,8 @@ const baseStyle = {
   overflowY: 'scroll',
   zIndex: 3,
   background: themeConfig.colors.light,
-  bottom: 'calc(var(--safe-area-inset-bottom) + var(--footer-height))',
+  color: themeConfig.colors.dark,
+  bottom: 'var(--footer-height)',
```

Comment:
it could make sense to add:
paddingBottom: var(--safe-area-inset-bottom)
right?

Reply:
No. `--footer-height` already includes this inset. Thats why iPhone X had issue with some gap above TabBar